core/src/errors.rs

```diff
@@ -159,3 +162,9 @@ impl From<MerkleBlockError> for BridgeError {
         BridgeError::MerkleBlockError(err)
     }
 }
+
+impl From<std::io::Error> for BridgeError {
```

Comment:
Let's not convert every std error to Database Error

Reply:
Done